src/devices/video/ppu2c0x_vt.h

```diff
@@ -26,6 +26,7 @@ enum vtxx_pal_mode {
 class ppu_vt03_device : public ppu2c0x_device {
 public:
 	ppu_vt03_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock = 0);
+	ppu_vt03_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, uint32_t clock = 0);
```

Comment:
Trampoline constructors should be protected, and the clock should never be defaulted in a trampoline constructor.

Also, all these PPUs should have their clocks documented - they definitely aren't clockless.  Defaulting the clocks to zero in the public constructors makes life easier but it hurts documentation,

Reply:
well I'll set them in the driver to use the PAL/NTSC clocks then, as there are no other XTALs on these.  Not 100% convinved that potentially bad documentation is better than just having at 0 (and this pattern is widespread in the existing NES code) but I'll try to keep the part I'm changing in check at least.